modules/apps/27-interchain-accounts/controller/keeper/params.go

```diff
@@ -0,0 +1,24 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/ibc-go/v2/modules/apps/27-interchain-accounts/controller/types"
+)
+
+// GetControllerEnabled retrieves the host enabled boolean from the paramstore
+func (k Keeper) GetControllerEnabled(ctx sdk.Context) bool {
+	var res bool
+	k.paramSpace.Get(ctx, types.KeyControllerEnabled, &res)
```

Comment:
Interesting API in the params module to pass a pointer instead of returning bytes. Is this purely to abstract the unmarshalling of types from the caller?

Reply:
No idea, but it is an interesting design decision. The reasoning for this decision is likely buried deep in the SDK (if ever documented)